src/content/perf/impeller.md

```diff
@@ -99,16 +99,15 @@ app's `Info.plist` file.
 
 ### Android
 
-As of Flutter 3.16, Impeller is available behind
-a flag on Android devices that support Vulkan.
-It continues to be in preview as of the 3.19 release.
+Impeller is available behind a flag on Android
+devices that support Vulkan.
 
 :::secondary Does your device support Vulkan?
 You can determine whether your Android device
 supports Vulkan at [checking for Vulkan support][vulkan].
 :::
 
-You can try Impeller on Vulkan-capable Android devices
+You can preview Impeller on Vulkan-capable Android devices
 by passing `--enable-impeller` to `flutter run`:
```

Comment:
**suggestion:** 
Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). 
Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
To preview Impeller on Vulkan-capable Android devices,
pass the `--enable-impeller` option to `flutter run`.
```